packages/@react-spectrum/accordion/docs/Accordion.mdx

```diff
@@ -56,6 +57,88 @@ import {Accordion, Disclosure, DisclosureHeader, DisclosurePanel} from '@react-s
 </Accordion>
 ```
 
+
+## Content
+
+Accordion accepts multiple [Disclosure](Disclosure.html) elements as children. Each disclosure accepts an `id` prop which is passed to the `onExpandedChange` handler to identify the expanded disclosure within the accordion. See the [Events](#events) section for more details about expansion.
+
+### Internationalization
+In order to internationalize an Accordion, all text content within the accordion should be localized.
+
+## Events
+
+Accordion accepts an `onExpandedChange` prop which is triggered when a disclosure is expanded or collapsed. The example below uses `onExpandedChange` to programmatically control disclosure expansion.
+
+```tsx example
+import {Key} from "@react-types/shared";
+
+function ControlledExpansion() {
+  let [expandedKeys, setExpandedKeys] = React.useState<Set<Key>>(new Set(['personal']))
+  
+  return (
+    <>
+      <Accordion expandedKeys={expandedKeys} onExpandedChange={setExpandedKeys}>
+        <Disclosure id="personal">
+          <DisclosureHeader>Personal Information</DisclosureHeader>
+          <DisclosurePanel>
+            <p>Personal information form here.</p>
+          </DisclosurePanel>
+        </Disclosure>
+        <Disclosure id="billing">
+          <DisclosureHeader>Billing Address</DisclosureHeader>
+          <DisclosurePanel>
+            <p>Billing address form here.</p>
+          </DisclosurePanel>
+        </Disclosure>
+      </Accordion>
+      <div style={{marginTop: '20px'}}>You have expanded: {expandedKeys}</div>
+    </>
+  )
+}
+```
+
 ## Props
 
 <PropTable component={docs.exports.Accordion} links={docs.links} />
+
+## Visual Options
+
+### Multi Selection
```

Comment:
I think instead of being under `Visual Options`, this can be under an `Expansion` section.